src/routes/configuration/environment-variables.mdx

```diff
@@ -27,8 +43,6 @@ console.log(import.meta.env.DB_PASSWORD); // undefined
 
 </Callout>
     
-2. In your Solid code, Vite will allow access to your environment variables by automatically injecting the environment variables into your application on compilation time.
-
 ```jsx
```

Comment:
This code block is disjointed now that the text above it has been removed - do we still need it?